tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsManager.cs

```diff
@@ -73,7 +73,9 @@ public async Task DisposeAsync()
     /// </summary>
     public PathwayContext? ExtractPathwayContext<TCarrier>(TCarrier headers)
         where TCarrier : IBinaryHeadersCollection
-        => IsEnabled ? DataStreamsContextPropagator.Instance.Extract(headers) : null;
+    {
+        return IsEnabled ? DataStreamsContextPropagator.Instance.Extract(headers) : null;
+    }
```

Comment:
Any reason to revert back to the older syntax? If not, do you mind reverting this change?

Reply:
Agreed, moved to newer syntax